Refit/RequestBuilderImplementation.cs

```diff
@@ -215,7 +215,17 @@ async Task AddMultipartItemAsync(MultipartFormDataContent multiPartContent, stri
             Exception e = null;
             try
             {
-                multiPartContent.Add(await settings.ContentSerializer.SerializeAsync(itemValue).ConfigureAwait(false), parameterName);
+                HttpContent cont;
+                if (settings.ContentSerializer is IContentSerializerWithCancellation serializerWithCancellation)
+                {
+                    cont = await serializerWithCancellation.SerializeAsync(itemValue).ConfigureAwait(false);
```

Comment:
This call is missing a cancellation token parameter

Reply:
Good catch, fixed it :)